docs/pages/modules/get-started.mdx

```diff
@@ -0,0 +1,131 @@
+---
+title: Get Started
+---
+
+import { CodeBlocksTable } from '~/components/plugins/CodeBlocksTable';
+import { Terminal } from '~/ui/components/Snippet';
+
+## Create a new module
+
+To create a new Expo module from scratch, just run the `create-expo-module` script as below. The script will ask you a few questions and then generate the native Expo module along with the example app for iOS and Android that uses your new module.
+
+<Terminal cmd={[`$ npx create-expo-module`]} />
+
+## Set up a library as an Expo module
+
+You may want to use the Expo Modules API in existing React Native libraries, for example with [AppDelegate Subscribers](./appdelegate-subscribers.mdx) you can hook into `AppDelegate` methods without requiring developers to copy any code over to their own `AppDelegate`. This is particularly useful to add seamless support for Expo managed projects to a library. The following steps will set up your existing React Native library to have access to the Expo Modules API.
+
+Create the [module config](./module-config.mdx) **expo-module.config.json** file just near your **package.json** and start from the empty object `{}` in there. We will fill it in later to enable specific features.
+The presence of the module config is enough for Expo Autolinking to recognize it as an Expo module and automatically link your native code.
```

Comment:
```suggestion
The presence of the module config is enough for [Expo Autolinking](https://docs.expo.dev/workflow/glossary-of-terms/#expo-autolinking) to recognize it as an Expo module and automatically link your native code.
```

I think it is good to add a link to our glossary to provide context on "Expo Autolinking"

Reply:
```suggestion
The presence of the module config is enough for Expo Autolinking](/workflow/glossary-of-terms/#expo-autolinking) to recognize it as an Expo module and automatically link your native code.
```

Good idea! Just remember to skip the domain in link.

Reply:
My bad 😅